geant3 fails to compile centos7

Hi, it started a few days ago, nothing special about the installation here are the relevant parts of the log:

+ cmake /home/mkrzewic/o2/sw/SOURCES/GEANT3/v2-5/v2-5 -DCMAKE_INSTALL_PREFIX=/home/mkrzewic/o2/sw/INSTALLROOT/c8c138df881dc801d5b519f646406e35a6adfaea/slc7_x86-64/GEANT3/v2-5-12 -DCMAKE_BUILD_TYPE=RELWITHDEBINFO -DCMAKE_SKIP_RPATH=TRUE
-- The C compiler identification is GNU 7.3.1                                                       
-- The CXX compiler identification is GNU 7.3.1                                                     
-- Check for working C compiler: /opt/rh/devtoolset-7/root/usr/bin/cc                               
-- Check for working C compiler: /opt/rh/devtoolset-7/root/usr/bin/cc -- works                      
-- Detecting C compiler ABI info                                                                    
-- Detecting C compiler ABI info - done                                                             
-- Detecting C compile features                                                                     
-- Detecting C compile features - done                                                              
-- Check for working CXX compiler: /opt/rh/devtoolset-7/root/usr/bin/c++                            
-- Check for working CXX compiler: /opt/rh/devtoolset-7/root/usr/bin/c++ -- works                   
-- Detecting CXX compiler ABI info                                                                  
-- Detecting CXX compiler ABI info - done                                                           
-- Detecting CXX compile features                                                                   
-- Detecting CXX compile features - done                                                            
-- The Fortran compiler identification is GNU 7.3.1                                                 
-- Check for working Fortran compiler: /opt/rh/devtoolset-7/root/usr/bin/gfortran                   
-- Check for working Fortran compiler: /opt/rh/devtoolset-7/root/usr/bin/gfortran  -- works         
-- Detecting Fortran compiler ABI info                                                              
-- Detecting Fortran compiler ABI info - done                                                       
-- Checking whether /opt/rh/devtoolset-7/root/usr/bin/gfortran supports Fortran 90                  
-- Checking whether /opt/rh/devtoolset-7/root/usr/bin/gfortran supports Fortran 90 -- yes           
-- Found ROOT 6.14/04 in /home/mkrzewic/o2/sw/slc7_x86-64/ROOT/v6-14-04-5                           
-- Configuring done                                                                                 
-- Generating done                                                                                  
-- Build files have been written to: /home/mkrzewic/o2/sw/BUILD/c8c138df881dc801d5b519f646406e35a6adfaea/GEANT3         
+ make -j 4 install                                                                                 


................

[ 47%] Building Fortran object CMakeFiles/geant321.dir/matx55/rsmt5t.F.o                            
[ 48%] Building Fortran object CMakeFiles/geant321.dir/matx55/ssmt5t.F.o                            
[ 48%] Building Fortran object CMakeFiles/geant321.dir/matx55/xmm55.F.o                             
[ 48%] Building Fortran object CMakeFiles/geant321.dir/miface/gfmdis.F.o                            
[ 48%] Building Fortran object CMakeFiles/geant321.dir/miface/gfmfin.F.o                            
[ 48%] Building Fortran object CMakeFiles/geant321.dir/miface/gmicap.F.o                            
/home/mkrzewic/o2/sw/SOURCES/GEANT3/v2-5/v2-5/geant321/gcbank.inc:29:19:                            
                                                                                                    
       EQUIVALENCE (Q(1),IQ(1),LQ(9)),(LQ(1),LMAIN),(IWS(1),WS(1))                                  
                   1                                                                                
Error: COMMON attribute conflicts with SAVE attribute in 'q' at (1)                                 
/home/mkrzewic/o2/sw/SOURCES/GEANT3/v2-5/v2-5/geant321/gcbank.inc:29:19:                            
                                                                                                    
       EQUIVALENCE (Q(1),IQ(1),LQ(9)),(LQ(1),LMAIN),(IWS(1),WS(1))                                  
                   1                                                                                
Error: COMMON attribute conflicts with SAVE attribute in 'q' at (1)                                 
/home/mkrzewic/o2/sw/SOURCES/GEANT3/v2-5/v2-5/geant321/gcbank.inc:29:19:                            
                                                                                                    
       EQUIVALENCE (Q(1),IQ(1),LQ(9)),(LQ(1),LMAIN),(IWS(1),WS(1))                                  
                   1                                                                                
Error: COMMON attribute conflicts with SAVE attribute in 'q' at (1)                                 
/home/mkrzewic/o2/sw/SOURCES/GEANT3/v2-5/v2-5/geant321/gcbank.inc:29:19:                            
                                                                                                    
       EQUIVALENCE (Q(1),IQ(1),LQ(9)),(LQ(1),LMAIN),(IWS(1),WS(1))                                  
                   1                                                                                
Error: COMMON attribute conflicts with SAVE attribute in 'q' at (1)                                 
/home/mkrzewic/o2/sw/SOURCES/GEANT3/v2-5/v2-5/geant321/gcbank.inc:29:19:                            
                                                                                                    
       EQUIVALENCE (Q(1),IQ(1),LQ(9)),(LQ(1),LMAIN),(IWS(1),WS(1))                                  
                   1                                                                                
Error: COMMON attribute conflicts with SAVE attribute in 'q' at (1)                                 
/home/mkrzewic/o2/sw/SOURCES/GEANT3/v2-5/v2-5/geant321/gcbank.inc:29:19:                            
                                                                                                    
       EQUIVALENCE (Q(1),IQ(1),LQ(9)),(LQ(1),LMAIN),(IWS(1),WS(1))                                  
                   1                                                                                
Error: COMMON attribute conflicts with SAVE attribute in 'q' at (1)                                 
/home/mkrzewic/o2/sw/SOURCES/GEANT3/v2-5/v2-5/geant321/gcbank.inc:29:19:                            
                                                                                                    
       EQUIVALENCE (Q(1),IQ(1),LQ(9)),(LQ(1),LMAIN),(IWS(1),WS(1))                                  
                   1                                                                                
.......................

Error: COMMON attribute conflicts with SAVE attribute in 'q' at (1)                                 
make[2]: *** [CMakeFiles/geant321.dir/build.make:7216: CMakeFiles/geant321.dir/miface/gmicap.F.o] Error 1               
make[2]: *** Waiting for unfinished jobs....                                                        
make[1]: *** [CMakeFiles/Makefile2:68: CMakeFiles/geant321.dir/all] Error 2                         
make: *** [Makefile:152: all] Error 2

--- a/geant3.sh
+++ b/geant3.sh
@@ -1,6 +1,6 @@
 package: GEANT3
 version: "%(tag_basename)s"
-tag: v2-5
+tag: v2-6
 requires:
   - ROOT
 build_requires:

